Source/Scene/Model.js

```diff
@@ -3293,6 +3304,32 @@ define([
         };
     }
 
+    function createClippingPlanesLengthFunction(model) {
+        return function() {
+            return model.clippingPlanes.length;
+        };
+    }
+
+    var scratchCartesian = new Cartesian3();
+    var scratchPlane = new Cartesian4();
+
+    function createClippingPlanesFunction(model, context) {
+        return function() {
+            var planes = model.clippingPlanes;
+            var length = planes.length;
+            var packedPlanes = new Array(length);
+            for (var i = 0; i < length; ++i) {
```

Comment:
This should probably be limited to 6. Check what happens if you have more than 6 clipping planes, there may be an error in setting the uniform.